elements/pf-popover/pf-popover.ts

```diff
@@ -360,7 +358,10 @@ export class PfPopover extends LitElement {
       <div id="container"
            style="${styleMap(styles)}"
            class="${classMap({ [anchor]: !!anchor, [alignment]: !!alignment })}">
-        <slot id="trigger" @keydown=${this.onKeydown} @click=${this.toggle}></slot>
+        <slot id="trigger"
+              @slotchange="${this.#triggerChanged}"
```

Comment:
@bennypowers `triggerChanged` is responsible for adding and removing event listeners to the element matching the id passed through the optional `trigger` property. Why do we need to call in the `slotchange` event handler for this slot?

Reply:
we want to use either the slotted trigger or the assigned one. and also, if a node gets removed from the slot or added back in, we need to hook up the listeners.

Reply:
we wouldn't want them to stay on the slot though if there's an assigned trigger :thinking: 

we might be able to handle that logic in the handler though